src/main/java/com/eprosima/fastdds/idl/templates/DDSPubSubTypeHeader.stg

```diff
@@ -243,11 +243,11 @@ private:
         return $struct.maxXCDRv1PlainTypeSerializedSize$ULL ==
                (detail::$struct.name$_offset_of<$struct.name$, detail::$struct.name$_f>() +
                sizeof($last(struct.members).typecode.cppTypename$));
-        $elseif(struct.inheritances)$
-        $if(last(struct.inheritances).isPlain)$
+        $elseif(struct.inheritance)$
+        $if(struct.inheritance.isPlain)$
```

Comment:
Join these conditional in only one. ST4 supports it.

Reply:
It has been checked that even though ST4 supports complex conditional expressions, if part of the expression evaluates to `null`, it does not break the conditional expression evaluation, so it can fail accessing a `null` instance.